domain/model/state/state.go

```diff
@@ -978,6 +978,145 @@ OR uuid IN (SELECT grant_on
 	return rval, nil
 }
 
+// ListModelSummariesForUser lists model summaries of all models the user has
+// access to. If no models are found then a nil slice is returned.
+// TODO(aflynn): 05-08-2024 - The ModelSummary struct includes a machine count,
+// unit count and cpu core count, model status as well as migration status. This
```

Comment:
I think we are storing machines now. Can we wire up this count ?

Reply:
From what I gather its still very much in flux, I think I'd like to wait until more is implemented and I can add machines and cpu cores and some of the other missing stuff at the same time.

I've added a task to the models epic for this

Reply:
Fair enough.